app/views/layouts/root.html.haml

```diff
@@ -2,21 +2,21 @@
   = yield
 
   = content_for :header do
-    %section.prerelease-govau--header.home
+    %section
+      //.prerelease-govau--header.home
```

Comment:
If we use the syntax `-#` instead of `//` the HAML won't be rendered as HTML comments, i.e.: 
`<!-- /.prerelease-govau--header.home -->`